benchmarks/inference/results/output_cuda_init.md

```diff
@@ -0,0 +1,12 @@
+| bs, compile | torch.load() / s | warmup latency / s | avg latency / s | max latency / s | min latency / s | throughput samples/s | GPU util % |
```

Comment:
I'm not convinced that having a folder with many .md files here is the right way to go to keep all results.
It is not really possible to easily compare two runs in this setting...

Reply:
Oh okay, would overwriting the existing file be better (since it would show a diff)?

Reply:
I think the best case would be that we have few enough metrics that we can have one experiment per-line and we grow that table down.
Or maybe make it a very large table and keep the important metrics on the left side so that github visualization shows them well.